### PR TITLE
Update sinusbot_installer.sh

### PR DESCRIPTION
**Pull request by DAThosting**
Fixed "fi"  problem

https://forum.sinusbot.com/threads/sinusbot-installer-script.1200/page-88#post-43209

### CODE REVIEW

File: sinusbot_installer.sh [hunk 1]
```diff
@@ -663,7 +663,7 @@ greenMessage "Packages installed"!
 if [[ $VIRTUALIZATION_TYPE == "openvz" ]]; then
   redMessage "You're using OpenVZ virtualization. You can't set your time, maybe it works but there is no guarantee. Skipping this part..."
 else
-  if [ -f /etc/centos-release ] || [ $(cat /etc/*release | grep "DISTRIB_ID=" | sed 's/DISTRIB_ID=//g') ]; then
+  if [[ -f /etc/centos-release ]] || [ $(cat /etc/*release | grep "DISTRIB_ID=" | sed 's/DISTRIB_ID=//g') ]; then
```
Comment (irgendwr) [inline sinusbot_installer.sh:666]: Is there a reason why you only do that on the first condition and not on the rest? (applies to multiple lines)

Comment (DAThosting) [reply sinusbot_installer.sh:666]: Only file checks are neccessary to use double square brackets.
All other condition may be break.

I tested this enhancement on Ubuntu 18.04, 19.04, Debian 9-10 and CentOS7 without problems.

**Review by patschi — APPROVED** [review]
As we use `bash` anyway, this changes are fine and we don't need to keep POSIX compatibility. While not tested myself, I don't think it breaks something.

**Comment by irgendwr** [comment]
Thanks :)

**Pull request merged**